static/sass/_pattern_cube-animation.scss

```diff
@@ -0,0 +1,70 @@
+$size: (
+  "width": 169,
+  "height": 195,
+);
+$speed: 2s;
+$positions: (
+  "1": (
+    "x": 100,
+    "y": 1,
+    "to": "6",
+  ),
+  "2": (
+    "x": 183,
+    "y": 49,
+    "to": "1",
+  ),
+  "3": (
+    "x": 183,
+    "y": 145.5,
+    "to": "4",
+  ),
+  "4": (
+    "x": 100,
+    "y": 193,
+    "to": "5",
+  ),
+  "5": (
+    "x": 16,
+    "y": 145.5,
+    "to": "2",
+  ),
+  "6": (
+    "x": 16,
+    "y": 49,
+    "to": "3",
+  ),
+);
+
+.cube {
+  background: url(https://assets.ubuntu.com/v1/c3137d5a-CUBE-2.svg);
+  background-size: map-get($size, "width") * 1px map-get($size, "height") * 1px;
+  height: map-get($size, "height") * 1px;
+  position: absolute;
+  transition: transform $speed;
+  width: map-get($size, "width") * 1px;
+}
+
+@each $index, $position in $positions {
+  $xPos: map-get($position, "x") * 1px;
+  $yPos: map-get($position, "y") * 1px;
+  .cube-#{$index} {
+    transform: translate(#{$xPos}, #{$yPos});
+  }
+}
+
+.hero-animation {
```

Comment:
`hero-animation` is too generic. Could you update it to something coupled to this animation. For example `.p-cube-animaiton`.